src/tsickle.ts

```diff
@@ -105,89 +105,177 @@ class ClosureRewriter extends Rewriter {
     super(file);
   }
 
-  emitFunctionType(fnDecl: ts.SignatureDeclaration, extraTags: jsdoc.Tag[] = []) {
+  /**
+   * Handles emittng the jsdoc for methods, including overloads.
+   * If overloaded, merges the signatures in the list of SignatureDeclarations into a single jsdoc.
+   * - Total number of parameters will be the maximum count found across all variants.
+   * - Any parameters beyond the minimum count found across all variants; prefix: "opt_"
+   * - Different names at the same parameter index will be joined with "_or_"
+   * - Variable args (...type[] in TypeScript) will be output as "...type",
+   *    except if found at the same index as another argument.
+   */
+  emitFunctionType(fnDecls: ts.SignatureDeclaration[], extraTags: jsdoc.Tag[] = []): string[] {
```

Comment:
Can you mention in the comment about what this returns?  It seems to be the parameter names?